apps/aecore/test/aecore_suite_utils.erl

```diff
@@ -344,11 +350,20 @@ spend(Node, FromPub, ToPub, Amount, Fee) ->
                fee          => Fee,
                nonce        => Nonce,
                payload      => <<"foo">>},
+
     {ok, Tx} = rpc:call(Node, aec_spend_tx, new, [Params]),
-    {ok, SignedTx} = rpc:call(Node, aec_keys, sign_tx, [Tx]),
+    {ok, SignedTx} = sign_on_node({dev1, Node}, Tx),
     ok = rpc:call(Node, aec_tx_pool, push, [SignedTx]),
     {ok, SignedTx}.
 
+sign_on_node({Id, Node}, Tx) ->
+    {_, {SignPrivKey, _SignPubKey}} = lists:keyfind(Id, 1, sign_keys()),
+    Bin = rpc:call(Node, aetx, serialize_to_binary, [Tx]),
```

Comment:
```suggestion
    aec_test_utils:sign_tx(Tx, SignPrivKey);
```

Reply:
This is already in another utils file :-)

Reply:
yes... 

![](https://media.giphy.com/media/eP1fobjusSbu/giphy.gif)